resources/lib/downloadutils.py

```diff
@@ -95,7 +95,7 @@ def getArtwork(self, data, art_type, parent=False, index=0, width=10000, height=
                 tagName = 'SeriesPrimaryImageTag'
                 idName = 'SeriesId'
             else:
-                tagName = 'Parent%sTag' % art_type
+                tagName = 'Parent%sImageTag' % art_type
```

Comment:
In my testing I found for example you are sending in things like "Thumb" so the tag was being set as ParentThumbTag instead of ParentThumbImageTag. Adding the "Image" to the text resolved that

Reply:
Yes you are correct